src/explore-education-statistics-common/src/modules/table-tool/components/TableSizeError.tsx

```diff
@@ -0,0 +1,77 @@
+import Button from '@common/components/Button';
+import ErrorPrefixPageTitle from '@common/components//ErrorPrefixPageTitle';
+import downloadService from '@common/services/downloadService';
+import { Subject } from '@common/services/tableBuilderService';
+import React, { useEffect, useRef } from 'react';
+
+interface Props {
+  focus: boolean;
+  releaseId?: string;
+  showDownloadOption?: boolean;
+  subject: Subject;
+}
+
+const TableSizeError = ({
+  focus = false,
+  releaseId,
+  showDownloadOption = true,
+  subject,
+}: Props) => {
+  const handleDownloadFile = async (fileId: string) => {
+    if (releaseId) {
+      await downloadService.downloadFiles(releaseId, [fileId]);
+    }
+  };
+
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (focus && ref.current) {
+      ref.current.focus();
+    }
+  }, [focus]);
+
+  return (
+    <div
```

Comment:
Seems like we could extract out a `BaseErrorSummary` (exported from the same file as `ErrorSummary`). It would be a dumb component that takes basic things like the `children`, `tiitle` and `id` and would be used by `ErrorSummary` itself.

We would then be able to re-use the `BaseErrorSummary` here instead of having to build another version of the component here.

Reply:
Happy that this has suggestion has been addressed